modules/seqexec/engine/src/main/scala/seqexec/engine/Engine.scala

```diff
@@ -172,10 +175,10 @@ class Engine[D, U](stateL: Engine.State[D]) {
 
   private def actionResume(id: Observation.Id, i: Int, cont: Stream[IO, Result])
   : HandleType[Unit] =
-    getS(id).flatMap(_.map { s =>
-    if (Sequence.State.isRunning(s) && s.current.execution.lift(i).exists(Action.paused))
-      modifyS(id)(_.start(i)) *> Handle.fromStream(act(id, (cont, i)))
-    else unit
+    getS(id).flatMap(_.collect {
+      case s@Sequence.State.Zipper(z, _)
```

Comment:
The other alternative is `Final`, but `actionResume` should never be called on a sequence in that state.